app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+from pathlib import Path
+import os
+
+
+def move_file(command: str) -> None:
+    command, old_file, new_file = command.split()
```

Comment:
Validate length of command too, it should be exactly 3 items

Reply:
and how to validate that string has 3 words without splitting it?